.github/workflows/continuous-integration.yaml

```diff
@@ -965,11 +986,8 @@ jobs:
           make setup_env
 
       - name: Set tag in env
-        # 'poetry version' cannot be piped properly so do it in 2 steps
-        # the project version does not have the leading v to be semver compatible
         run: |
-          PROJECT_VERSION=$(poetry version)
-          PROJECT_VERSION=$(echo "$PROJECT_VERSION" | cut -d ' ' -f 2)
+          PROJECT_VERSION=$(poetry version --short)
```

Comment:
why the `--short`?

Reply:
it's because `poetry --version` outputs "concrete-ml 1.1.0" while the `--short` option only gives the version ("1.1.0")